core/src/main/resources/hudson/model/View/sidepanel_lt.properties

```diff
@@ -1,29 +1,9 @@
-# The MIT License
-#
-# Copyright (c) 2004-2010, Sun Microsystems, Inc.
```

Comment:
I doubt the header is valid, because all Jenkins contributors from Sun Lithuanian IIRC. So I agree with its removal